pkgs/development/python-modules/backgroundremover/default.nix

```diff
@@ -0,0 +1,53 @@
+{ buildPythonPackage, fetchPypi, lib, certifi, charset-normalizer, ffmpeg-python
+, filelock, filetype, hsh, idna, more-itertools, moviepy, numpy, pillow
+, pymatting, pysocks, requests, scikitimage, scipy, six, torch, torchvision
+, tqdm, urllib3, waitress, gdown }:
+buildPythonPackage rec {
+  pname = "backgroundremover";
+  name = pname;
```

Comment:
remove name, use pname only

name is calculated as "${pname}-${version}" automatically